src/lru.rs

```diff
@@ -322,11 +327,8 @@ fn _expired(timestamp: Timestamp, now: Timestamp) -> bool {
 pub fn compute_weight<K: HasWeight, V: HasWeight>(key: &K, value: &V) -> Weight {
     // this isn't perfect because it ignores some hashtable and btreeset
     // overhead, but it's a pretty good guess at the memory usage of an entry
-    (3*key.weight()
-     + value.weight()
-     + mem::size_of::<Weight>()
-     + mem::size_of::<Timestamp>()
-     + mem::size_of::<Option<Timestamp>>())
+    (3 * key.weight() + value.weight() + mem::size_of::<Weight>() + mem::size_of::<Timestamp>() +
+     mem::size_of::<Option<Timestamp>>())
```

Comment:
this probably needs to be a series of `+=` lines or something if this is what the formatter will do to it. Otherwise the important stuff is all off of the screen, which IME is where the bugs lurk.

Does Rust not traditionally do the 80 columns thing? I see a number of places where I tried to keep to it but rustfmt doesn't agree